### DISCUSSION TITLE
How to mouse events?

### DISCUSSION THREAD

**Discussion by kezara** [canton7/Stylet#316] [Q&A] [answered]
How to handle mouse events, left or right button down/up/click, mouse enter, mouse leave etc.

Many thanks in advance.

Cheers

**Comment by canton7** [comment, marked as answer]
Use the normal WPF events. You can attach actions to these, e.g. `MouseEnter="{s:Action ...}"`

**Reply by kezara** [reply]
thanks, I hurried a little, since I've done it, in the meantime, like you said
